apps/arweave/src/ar_global_sync_record.erl

```diff
@@ -74,15 +74,14 @@ init([]) ->
 	SyncRecord =
 		lists:foldl(
 			fun(Module, Acc) ->
-				StoreID =
-					case Module of
-						"default" ->
-							"default";
-						_ ->
-							ar_storage_module:id(Module)
-					end,
-				R = ar_sync_record:get(ar_data_sync, StoreID),
-				ar_intervals:union(R, Acc)
+				case Module of
+					"default" -> ar_intervals:union(ar_sync_record:get(ar_data_sync, "default"), Acc);
+					%% Ignore replica.2.9 packing. This is a temporary solution until
+					%% we can support data syncing in batches corresponding to the
+					%% replica.2.9 entropy footprint
+					{_, _, {replica_2_9, _}} -> Acc;
+					_ -> ar_intervals:union(ar_sync_record:get(ar_data_sync, ar_storage_module:id(Module)), Acc)
```

Comment:
It'd be good to remove the get/union duplication. Maybe keep the old StoreID case statement and then do the new replica_2_9 case?

alternatively maybe `ar_storage_module:id` can be updated to handle the `"default"` module